### PR TITLE
Rebuild for Python 3.10 and pytorch 1.11

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
This requires https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/374

**Review by baszalmstra — APPROVED** [review]

**Comment by xhochy** [comment]
@conda-forge-admin please restart ci

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -2,3 +2,4 @@ pytorch:
   - 1.8.*
   - 1.9.*
   - 1.10.*
+  - 1.11.*
```
Comment (h-vetinari) [inline recipe/conda_build_config.yaml:5]: Why are we still building for so many pytorch versions here (and also missing the last recent two)...?

**Review by h-vetinari — COMMENTED** [review]
The failures all look like intermittent errors that just need a restart.

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (xhochy) [reply recipe/conda_build_config.yaml:5]: This is detetectrons officially supported versions of the last release. let's build for the other two and get rid of the matrix completely with the pytorch113 migration

Comment (xhochy) [inline recipe/conda_build_config.yaml:5]:
```suggestion
  - 1.11.*
  - 1.12.*
```

**Comment by xhochy** [comment]
@conda-forge-admin please rerender

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin please rerender

**Pull request merged**